it/google-cloud-platform/src/main/java/com/google/cloud/teleport/it/gcp/LoadTestBase.java

```diff
@@ -67,6 +70,9 @@ public abstract class LoadTestBase {
   private static final double PD_SSD_PER_GB_HR = 0.000298;
   private static final double SHUFFLE_PER_GB_BATCH = 0.011;
   private static final double SHUFFLE_PER_GB_STREAMING = 0.018;
+  private static final Pattern WORKER_START_PATTERN = Pattern.compile("^Starting \\d+ workers.*$");
```

Comment:
The worker harness still has >2 min start time after this log. Checked an example pipeline:

Start time May 23, 2023 at 10:25:27 PM GMT-4 <-- start time

2023-05-23 22:25:43.906 EDT Starting 5 workers in us-central1-a... <-- this log

There is a debug level log:

2023-05-23 22:27:21.277 EDT
All workers have finished the startup processes and began to receive work requests.

indicates actual startup finished. This log might be a better candidate

Same log found for runner v2 job

Reply:
Thanks! Changed the log